README.md

```diff
@@ -75,7 +78,9 @@ This block describes the configuration for check1.
   "cleanup-time-unit": "SECONDS",
   "max-event-batch-content-size": "1048576",
   "rule-execution-timeout": 5000,
-  "auto-silence-check-after-sequence-rule": false
+  "auto-silence-check-after-sequence-rule": false,
+  "time-precision": "PT0.000000001S",
```

Comment:
Please, add a description for format in which the `time-precision` should be specified